src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/OffenderService.kt

```diff
@@ -50,6 +82,27 @@ OffenderService(
     }
   }
 
+  fun isXWorkingDaysBefore(days: Int, releaseDate: LocalDate?): Boolean {
+    if (releaseDate === null) return false
+    val dateBeforeXWorkingDays = getEarliestReleaseDate(days, releaseDate)
+    return LocalDate.now(clock).isEqual(dateBeforeXWorkingDays) || LocalDate.now(clock).isAfter(dateBeforeXWorkingDays)
+  }
+
+  private fun isEligibleForEarlyRelease(releaseDate: LocalDate?): Boolean {
```

Comment:
this has different logic to the other criteria... we should only have one definition of what the criteria for early release is.. this presumably is the criteria for something else?